config/settings/base.py

```diff
@@ -509,6 +511,8 @@ def env(name, default=NoDefaultValue, type_=str):
 # New feature branch prefix:
 BRANCH_PREFIX = env("BRANCH_PREFIX", default=None)
 
+ENABLE_WALKTHROUGHS = env("ENABLE_WALKTHROUGHS", default=True, type_=boolish)
```

Comment:
Don't bother changing it, but when we're putting a new feature behind a feature flag, it should probably be off by default.

Reply:
Makes sense. I went this way because I'm not used to using feature flags in this project and I was thinking of it as a temp workaround while we flesh out the other walkthroughs -- but I think you're right. 👍